pkg/fuzz/component/path.go

```diff
@@ -83,3 +83,11 @@ func (q *Path) Rebuild() (*retryablehttp.Request, error) {
 	}
 	return cloned, nil
 }
+
+// Clones current state to a new component
+func (q *Path) Clone() Component {
+	return &Path{
+		value: q.value.Clone(),
+		req:   q.req,
```

Comment:
I guess that if we are fuzzing the same request we would need a clone here as well?

Reply:
yup , missed that one